### DISCUSSION TITLE
Adding a custom module to Caddy build

### DISCUSSION THREAD

**Discussion by theoww** [dunglas/symfony-docker#483] [Q&A] [answered]
Hello,

I'd like to know if there is a way to add a custom module to the caddy build since FrankenPHP came out.
I searched some docs about it but i couldn't find.

On the previous versions i used the following instructions in my Dockerfile:

```
RUN xcaddy build \
	--with github.com/dunglas/mercure \
	--with github.com/dunglas/mercure/caddy \
	--with github.com/dunglas/vulcain \
	--with github.com/dunglas/vulcain/caddy \
        --with github.com/caddy-dns/ovh <-- The custom module i want to add
```

Thank you !

**Comment by maxhelias** [comment, marked as answer]
Hello,

You need to check out FrankenPHP : https://github.com/dunglas/frankenphp/blob/main/docs/docker.md#how-to-install-more-caddy-modules
